chainer_chemistry/models/gin.py

```diff
@@ -28,10 +29,12 @@ class GIN(chainer.Chain):
         n_edge_types (int): number of edge type.
             Defaults to 4 for single, double, triple and aromatic bond.
     """
-    def __init__(self, out_dim, hidden_channels=16,
+
+    def __init__(self, out_dim, node_embedding=False, hidden_channels=16,
+                 out_channels=None,
                  n_update_layers=4, n_atom_types=MAX_ATOMIC_NUM,
                  dropout_ratio=0.5, concat_hidden=False,
-                 weight_tying=True, activation=functions.identity,
+                 weight_tying=False, activation=functions.identity,
```

Comment:
not change default value

Reply:
weight_tying has a bug and I think it should be False by default

Reply:
#399